### ISSUE TITLE
I can't train....the loss diverges....how do I solve this?

### ISSUE THREAD

**Issue by zeek-han** [MahdiHajibabaei/unified-embedding#6]
I train ResNet-20.prototxt, but the loss diverges.
I checked the paper, and there is no configuration difference the paper and the prototxt.
Please, tell me how I solve this proble,
Thank you

**Comment by MahdiHajibabaei** [comment]
Hi,

If you have done the parsing and allocation of files to each ID right, decreasing the initial learning rate by 10 percent should solve the problem (for example decrease it from 0.05 to 0.045). We selected the largest learning rate that did not result in divergence and given that the dataset might have changed since we ran the experiments, the largest value of learning rate might have changed as well.
